workers/unity/Packages/io.improbable.gdk.playerlifecycle/Systems/PlayerCreation/HandleCreatePlayerRequestSystem.cs

```diff
@@ -11,11 +11,13 @@ namespace Improbable.Gdk.PlayerLifecycle
     public class HandleCreatePlayerRequestSystem : ComponentSystem
     {
         private CommandSystem commandSystem;
+        private EntityReservationSystem entityReservationSystem;
 
         protected override void OnCreate()
         {
             base.OnCreate();
             commandSystem = World.GetExistingSystem<CommandSystem>();
+            entityReservationSystem = World.GetOrCreateSystem<EntityReservationSystem>();
```

Comment:
If we are `GetOrCreate` here, do we need the system to be added in the helper? Or should this be changed to `GetExistingSystem()`?

Reply:
imo should be `GetExistingSystem` - if it's not there then I think something has gone wrong when setting up the server systems